### DISCUSSION TITLE
debug:translation shows 'Missing' for translation messages which are already defined in bundles

### DISCUSSION THREAD

**Discussion by JaydeepGalitein** [symfony/symfony#47301] [Q&A]
While working with symfony translations, I have overwritten a template file from one bundle and used the translation messages in project. Translation messages are defined very well according to domain (inside `messages.en.xlf`) in bundle and works fine.

But when I run `php bin/console debug:translation en --only-missing`, those messages are reported missing. Whenever I use any translation message outside the bundle, It reports missing error for those translation messages.

I have also tried with `php bin/console debug:translation en --only-missing BundleName` which works fine without showing any missing errors.

1. Why is `php bin/console debug:translation en --only-missing` reporting these defined translation messages as **Missing**?
2. Is there any possible way by which all the defined translation messages can be tracked using `php bin/console debug:translation` without any **Missing** errors?

**Comment by MatTheCat** [comment]
Would the `--all` option fit your use case?

https://github.com/symfony/symfony/blob/4c438af085fb674fd37f3bd1269318d9759ee55c/src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php#L88

**Reply by JaydeepGalitein** [reply]
@MatTheCat Hi Mat,

Thank you for your suggestion. I think I have found the solution and I have described it in detail [here](https://github.com/symfony/symfony/issues/47562#issuecomment-1250825968)
